CHANGELOG.md

```diff
@@ -87,6 +87,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* [\#9780](https://github.com/cosmos/cosmos-sdk/pull/9780) Remove gogoproto `moretags` YAML annotations and add `sigs.k8s.io/yaml` for YAML marshalling.
```

Comment:
Could you add an entry in the CLI-breaking change? Something along the lines of "Use sigs.k8s.io for yaml, which might lead to minor YAML output changes".